### ISSUE TITLE
In file included from v8go.cc:3:

### ISSUE THREAD

**Issue by 1920853199** [rogchap/v8go#27]
```
# rogchap.com/v8go
In file included from v8go.cc:3:
D:/wwwroot/go/src/rogchap.com/v8go/deps/include/v8.h:7161:5: error: typedef 'v8::UnhandledExceptionCallback' is initialized (use decltype instead)
     _EXCEPTION_POINTERS* exception_pointers);
     ^~~~~~~~~~~~~~~~~~~
D:/wwwroot/go/src/rogchap.com/v8go/deps/include/v8.h:7161:5: error: '_EXCEPTION_POINTERS' was not declared in this scope
D:/wwwroot/go/src/rogchap.com/v8go/deps/include/v8.h:7161:26: error: 'exception_pointers' was not declared in this scope
     _EXCEPTION_POINTERS* exception_pointers);
                          ^~~~~~~~~~~~~~~~~~
D:/wwwroot/go/src/rogchap.com/v8go/deps/include/v8.h:7161:26: note: suggested alternative: '_pxcptinfoptrs'
     _EXCEPTION_POINTERS* exception_pointers);
                          ^~~~~~~~~~~~~~~~~~
                          _pxcptinfoptrs
D:/wwwroot/go/src/rogchap.com/v8go/deps/include/v8.h:8844:7: error: 'UnhandledExceptionCallback' has not been declared
       UnhandledExceptionCallback unhandled_exception_callback);
       ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by rogchap** [comment]
Does not support windows: see #23

**Comment by tmc** [comment]
@rogchap can we change the title here and/or close this issue?

**Comment by neptoess** [comment]
Fixed by #46
